Piccy/Piccy/View Controllers/Login and Register/RegistrationTableViewController.m

```diff
@@ -18,6 +18,16 @@ @interface RegistrationTableViewController ()
 @property (weak, nonatomic) IBOutlet UITextField *phoneNumberField;
 @property (weak, nonatomic) IBOutlet UITextField *dateOfBirthField;
 
+@property (nonatomic) bool usernameTaken;
+@property (nonatomic) bool usernameTooShort;
+@property (nonatomic) bool usernameHasWeirdCharacters;
+@property (nonatomic) bool passwordsDontMatch;
+@property (nonatomic) bool passwordTooShort;
```

Comment:
one thing you can make your pr more clear is only add things you are using in this commit. For example, you are not using this passwordTooShort in this commit, but you added this property here.